swagger.json

```diff
@@ -1519,6 +1573,52 @@
                 }
             }
         },
+        "/fleet/hos/audit_logs": {
```

Comment:
This should be at `/fleet/drivers/hos/audit_logs` for consistency with `/fleet/drivers/:id/hos/audit_logs`.

Reply:
good call, I think this change is fine to make as we are making a change to the overall response as well.